bleep-core/src/scala/bleep/commands/Test.scala

```diff
@@ -18,7 +22,22 @@ case class Test(watch: Boolean, projects: Array[model.CrossProjectName]) extends
   override def runWithServer(started: Started, bloop: BuildServer): Either[BleepException, Unit] =
     DoSourceGen(started, bloop, watchableProjects(started)).flatMap { _ =>
       val targets = BleepCommandRemote.buildTargets(started.buildPaths, projects)
-      val result = bloop.buildTargetTest(new bsp4j.TestParams(targets)).get()
+      val tests = bloop.buildTargetScalaTestClasses(new bsp4j.ScalaTestClassesParams(targets)).get()
```

Comment:
This is deprecated, but I am not quite sure how the JVM replacement works. I will look into it.

Reply:
this has been that way for a while. we can do this in a follow-up I think